src/main/java/edu/ie3/datamodel/models/input/container/SystemParticipants.java

```diff
@@ -124,6 +146,10 @@ public void add(EvcsInput evcsInput) {
     evCS.add(evcsInput);
   }
 
+  public void add(EvInput evInput) {
```

Comment:
Do we still want to have `add` methods?

Reply:
Looks like ... I removed them - if you not agree with this let me know